spec/models/user_spec.rb

```diff
@@ -519,12 +519,39 @@
     end
   end
 
+  describe '#gift_sticker' do
```

Comment:
Looks like this description is out of date.
```suggestion
  describe '#gift' do
```